### ISSUE TITLE
Spotify text shows "error" for a week.

### ISSUE THREAD

**Issue by alperdincer** [Toxblh/MTMR#157]
I've been using MTMR for awhile and I got an issue with Spotify part for a week. I got "error" text on Spotify part but the buttons are working. I only don't see the song title. I've updated the MTMR after this happened, but nothing changed. 

What is your suggestion(s)?

**Comment by BoKKeR** [comment]
it works for me

**Comment by fontvu** [comment]
> it works for me, did you install spotify using homebrew?

Yes, what's wrong with it? I was running the "brew cu -a -f" and it crashed while upgrading Spotify. After running "brew cask reinstall spotify", the Spotify part in preset was broken.
